docs/wiki/The library - Module design.md

```diff
@@ -273,8 +273,9 @@ param diagnosticEventHubAuthorizationRuleId string = ''
 @description('Optional. Name of the diagnostic event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.')
 param diagnosticEventHubName string = ''
 
-@description('Optional. The name of logs that will be streamed.')
+@description('Optional. The name of logs that will be streamed. "allLogs" includes all possible logs for the resource.')
 @allowed([
+  'allLogs'
```

Comment:
Sorry I can't comment the right line. In line 282 below it should be `'allLogs'` instead of `<LogsIfAny>`